### ISSUE TITLE
Утечки памяти.

### ISSUE THREAD

**Issue by dwap** [alexander-akhmetov/python-telegram#5]
Посмотрел пакет
постоянно растут
self._results
self._queue

и ошибка при удалении
    def stop(self) -> None:
        self. _tdjson._td_json_client_destroy(self.td_json_client)

**Comment by alexander-akhmetov** [comment]
Да, очереди (`Telegram._queue`, `Telegram. _workers_queue`, `BaseWorker._queue`) сейчас не ограничены, `maxsize` не передается (https://github.com/alexander-akhmetov/python-telegram/issues/6).

> и ошибка при удалении

Какая именно?

**Comment by dwap** [comment]
> Какая именно?

Вообще остановка tdlib не вызывается.
Т.е. если сделать удаление 'del tg', то при повторном создании создании, будет таймаут:
RuntimeError: Telegram error: {'@type': 'error', 'code': 400, 'message': "Can't lock file because it is already in use; check for another program instance running", '@extra': {'request_id': 'updateAuthorizationState'}}
процесс остался

если принудительно, то
AttributeError: function '_td_json_client_destroy' not found

**Comment by dwap** [comment]
По утечкам 
Закомментировал
https://github.com/alexander-akhmetov/python-telegram/blob/963e99853bb59dd6bec1f49c20cde76f0c7c45a0/telegram/client.py#L240

После
https://github.com/alexander-akhmetov/python-telegram/blob/963e99853bb59dd6bec1f49c20cde76f0c7c45a0/telegram/client.py#L261
Добавил
            async_result._parse_update(update)
            self._results.pop(request_id)
